config/runtime/tests/ltp.jinja2

```diff
@@ -0,0 +1,15 @@
+
+- test:
+    name: {{ node.name }}
+    timeout:
+      minutes: {{ job_timeout }}
```

Comment:
Please provide a default timeout value here

Reply:
Every LTP test specifies job_timeout. It is configurable on the test's end. That is why job_timeout is being used here.

Reply:
@a-wai Am I missing something?

Reply:
I understood from your other comment that we need to specify the default timeout value here. Updated.
> job_timeout should be an int (and the default value should actually be in the template IMHO).